process/project_proposals.adoc

```diff
@@ -19,6 +20,7 @@
  .. release methodology and mechanics
  .. social media accounts
  .. community size and any existing sponsorship
+ .. who is currently known to be using the project? Are they using it in production and at what scale? (It may be hard to obtain accurate data for this, but any supporting evidence of usage is helpful)
```

Comment:
Note that one of the primary motivations behind Sandbox was to provide a neutral home to facilitate early-stage multi-company collaboration efforts, even for cases where those efforts have not yet reached a stage of being used. See https://www.cncf.io/sandbox-projects/ for more detail.

How do we propose to accommodate this goal? I assume that "signs of being used" would be considered "good to have" but not a blocker for entry into the Sandbox?